Dockerfile

```diff
@@ -0,0 +1,29 @@
+#Deriving the latest base image
+FROM python:latest
+
+
+#Labels as key value pair
+LABEL Maintainer="gpa.me17"
+
+
+# Any working directory can be chosen as per choice like '/' or '/home' etc
+# i have chosen /usr/app/src
+WORKDIR /usr/app
+
+#to COPY the remote file at working directory in container
+COPY entry.py ./
+COPY src ./src
+COPY data ./data
+COPY results ./result
+COPY test ./test
+# Now the structure looks like this '/usr/app/src/test.py'
+
+# COPY requirements.txt requirements.txt
+# RUN pip install -r requirements.txt
+
+
+
+#CMD instruction should be used to run the software
+#contained by your image, along with any arguments.
+
+CMD [ "python", "./entry.py"]
```

Comment:
How does this differ from using `ENTRYPOINT` instead of `CMD`?

Reply:
Changed to ENTRYPOINT since this is recommended when building an executable Docker image using commands that always need to be executed.
CMD instructions are best for an additional set of arguments that act as default instructions till there is an explicit command line usage when a Docker container runs.